pkg/domain/entities/volumes.go

```diff
@@ -59,6 +59,55 @@ type VolumeConfigResponse struct {
 	Anonymous bool `json:"Anonymous"`
 }
 
+// swagger:model VolumeInfo
+type VolumeInfo struct {
+
+	// Date/Time the volume was created.
+	CreatedAt string `json:"CreatedAt,omitempty"`
+
+	// Name of the volume driver used by the volume.
+	// Required: true
+	Driver string `json:"Driver"`
+
+	// User-defined key/value metadata.
+	// Required: true
+	Labels map[string]string `json:"Labels"`
```

Comment:
This is optional

Reply:
Actually, now that I think about it... this is returned by the server, so required/optional might not make sense.

"Always included" vs "Optional" maybe?